src/Pages/Welcome/Welcome.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react'
+import './Welcome.css'
+
+const Welcome = () => {
+    const [userName, setUserName] = useState();
+
+    const continueHandler = () => {
+        localStorage.setItem("userName", userName);
+        window.location.reload(false);
```

Comment:
what is it use here

Reply:
It sets username to local storage and reload the app after clicking